teps/0096-pipelines-v1-api.md

```diff
@@ -250,6 +251,8 @@ This policy should be updated to include Tekton metrics as part of the API. No o
     independently of Pipelines V1. Merging this TEP as implementable, or any features listed in this TEP, aren't V1 blockers.
     While we aim to make the transition to V1 as smooth as possible for PipelineResource users, the focus of V1 is to
     stabilize existing features, and we don't want to block a V1 API on new, experimental features.
+- Deprecate `taskRun.spec.taskRef.bundle` and `pipelineRun.spec.pipelineRef.bundle`, as these features will be replaced by
+  the ["bundle" resolver](https://github.com/tektoncd/resolution/tree/main/bundleresolver).
```

Comment:
would the `bundleresolve` made part of the `pipeline` controller?

I don't have a use case for bundles at the moment but trying to understand how would this impacts the users which might be using `bundles` without any other controller?

Reply:
Great point! I think it would make sense to bring the bundle resolver into Tekton by default to replace the OCI bundles feature. thoughts @chitrangpatel @vdemeester @abayer? relevant to https://github.com/tektoncd/pipeline/issues/4710

Reply:
@pritidesai @lbernick yes, that would be a requirement for us removing the `taskref.bundle`. `tektoncd/resoluton` is meant to be brought back into `tektoncd/pipeline` *and* some of the resolver should be built-in (cluster, git and bundle at least).

Reply:
Updated the TEP to specify that we will bring remote resolution + the bundle resolver into Pipelines.